### PR TITLE
Created a Dockerfile

### LINKED ISSUE #2: Dockerize the Project for Easy Setup
**Is your feature request related to a problem? Please describe.**
I think dockerizing the whole project will help us to run and set up the project in an easy way

**Describe the solution you'd like**
Create a Dockerfile, to automate the process.

### PR DESCRIPTION
**Pull request by iamkrati22**
this fixes #2  This Dockerfile contains all the required dependencies to run on the local machine as per the requirements.txt

### CODE REVIEW

**Comment by paramsiddharth** [comment]
Please follow GitHub contribution guidelines when making a PR: https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue
Use `close` or `closes` to automatically close the issue related to the PR.

**Comment by DARK-art108** [comment]
@paramsiddharth take a look over the PR.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM python:3.9
+
+WORKDIR /app
+
+COPY bot.py .
+COPY requirements.txt .
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+CMD ["python", "./bot.py"]
```
Comment (paramsiddharth) [inline Dockerfile]: Please add a newline at the end of file. This is indicated by a image in the PR info.

**Review by paramsiddharth — CHANGES_REQUESTED** [review]
Good job! Just a minor change to do.
Also, I would really encourage you to use Git instead of uploading to GitHub.

**Comment by paramsiddharth** [comment]
Looks good to me. Merged!

**Pull request merged**